gef.py

```diff
@@ -1854,6 +1854,13 @@ def mprotect_asm(cls, addr, size, perm):
         return "; ".join(insns)
 
 
+class ARM_M(ARM):
+    arch = "ARM-M"
```

Comment:
I have a curious issue with blank characters here: `SyntaxError: invalid character '￼' (U+FFFC)`. Maybe a github artefact.

Reply:
The commit seems to be fine:
```
$ git show | tail -n 26 | head -n 5 | hexdump -C
00000000  2b 63 6c 61 73 73 20 41  52 4d 5f 4d 28 41 52 4d  |+class ARM_M(ARM|
00000010  29 3a 0a 2b 20 20 20 20  61 72 63 68 20 3d 20 22  |):.+    arch = "|
00000020  41 52 4d 2d 4d 22 0a 2b  0a 2b 20 20 20 20 61 6c  |ARM-M".+.+    al|
00000030  6c 5f 72 65 67 69 73 74  65 72 73 20 3d 20 41 52  |l_registers = AR|
00000040  4d 2e 61 6c 6c 5f 72 65  67 69 73 74 65 72 73 5b  |M.all_registers[|
00000050  3a 2d 31 5d 20 2b 20 5b  22 24 78 70 73 72 22 2c  |:-1] + ["$xpsr",|
00000060  5d 0a 2b 20 20 20 20 66  6c 61 67 5f 72 65 67 69  |].+    flag_regi|
00000070  73 74 65 72 20 3d 20 22  24 78 70 73 72 22 0a     |ster = "$xpsr".|
```